src/v3/src/components/InfoBox/InfoBox.tsx

```diff
@@ -56,7 +56,13 @@ const InfoBox: UISchemaElementComponent<{
         {...({ 'data-se': dataSe })}
         className={`infobox-${messageClass.toLowerCase()}`}
       >
-        { title && <AlertTitle>{title}</AlertTitle> }
+        { title && (
+        <AlertTitle
+          role="heading"
```

Comment:
Is there a way to use a semantic tag instead of applying an aria attr? They should generally be used as a last-resort. Do we need to talk to Odyssey?

Reply:
This is from MUI unfortunately, not Odyssey. It looks like MUI uses a `div` with styling for this rather than use a semantic heading element.

![image](https://user-images.githubusercontent.com/72248639/215162512-6e9ab6b7-e914-488e-bd9e-1dadc3341752.png)

Reply:
See also this hidden thread: https://github.com/okta/okta-signin-widget/pull/3040#discussion_r1086209062

Reply:
`role="heading"` is no longer needed and has been removed.  `AlertTitle` should already have the necessary props built in.

Reply:
ah yeah thats why the `role="heading"` was needed since MUI doesnt use a semantic title

Reply:
I suppose an alternative would be to use the Typography component?

Reply:
Does it look OK if we use Typography? I wanted to use AlertTitle since it seems to be tailor made for the Alert component but if it isn't suiting our needs we can look at the other components.